kiwi/filesystem/base.py

```diff
@@ -316,6 +319,28 @@ def set_property_readonly_root(self) -> None:
         """
         raise NotImplementedError
 
+    def _generate_seed_uuid(self, label: str, random_bits: int = 128) -> str:
+        """
+        Create random UUID. If SOURCE_DATE_EPOCH is present use
+        SOURCE_DATE_EPOCH + label name as seed
+        """
+        sde = os.environ.get('SOURCE_DATE_EPOCH')
```

Comment:
I think it'd be a good idea to allow a command-line option to set this too. If `SOURCE_DATE_EPOCH` doesn't exist, I think it's worth exposing a CLI knob to provide a timestamp seed.

Reply:
yeah let's allow that